fh-messaging/npm-shrinkwrap.json

```diff
@@ -91,7 +91,7 @@
             "iconv-lite": {
               "version": "0.4.13",
               "from": "iconv-lite@0.4.13",
-              "resolved": "https://registry.npmjs.org/iconv-lite/-/iconv-lite-0.4.13.tgz"
+              "resolved": "http://npm.skunkhenry.com/iconv-lite/-/iconv-lite-0.4.13.tgz"
```

Comment:
@luigizuccarelli Not sure why the url would change to skunkhenry here (and in other places below). 🤔 It's probably fine, but it might have something to do with your local ~/.npmrc.